### PR TITLE
Added Javascript Tutorial 6 Text 

### PR DESCRIPTION
**Pull request by SHeimlich**
Also updated tutorial 5's arc's manifest file.

### CODE REVIEW

File: particles/Tutorial/Javascript/README.md [hunk 1]
```diff
@@ -369,3 +369,80 @@ defineParticle(({SimpleParticle, html}) => {
 ```
 
 When you execute this recipe, you should see everyone being greeted. If you added or remove an entity from the `PeopleData` store, the number of people greeted will change accordingly thanks to the template interpolation.
+
+# The Lore of a JSON Store
+
+Hopefully by now you are starting to see how the different components of Arcs work together to preserve user sovereignty as you develop. This is the final chapter in our introductory tutorials which are designed to introduce the base Arcs concepts. But don’t worry, this is not the end of our tutorials! From here you can continue to the next set of tutorials where you’ll build a tic-tac-toe game with interchangeable human and computer players.
+
+Alright, let’s get to the actual tutorial! This time, instead of having our store be internal to the Arcs Manifest file, we’re going to use an external store. In our case, this will be a JSON document. Let’s start with the Arcs Manifest file:
```
Comment (csilvestrini) [inline particles/Tutorial/Javascript/README.md]: Can you rephrase "external store"? That name has another, more complex, meaning (we sometimes talk about stores that get mapped into a recipe from the main context as an external store).

It's also maybe a subtle point, but it's not really the "store" that is internal to the Arcs Manifest file, it's the store's data.

Here's what I would suggest:

```
This time, instead of embedding the store's data directly inside
the Arcs Manifest file (as a `resource`), we're going to load it
from a separate JSON file.
```

File: particles/Tutorial/Javascript/README.md [hunk 2]
```diff
@@ -369,3 +369,80 @@ defineParticle(({SimpleParticle, html}) => {
...
+});
+```
+
+
+Before we finish, let’s go over all of the concepts we have introduced in these tutorials.
+
+>- *Particle* - Modular component of functionality. Ideally small units so particles can be reusable. 
+>- *Recipe* - A combination of particles to create features and systems.
+>- *Template Interpolation* - A mechanism to substitute formatted data into renderable elements.
+>- *Slots* - an element of the Arcs platform that allows particles to render on a user interface.
```
Comment (csilvestrini) [inline particles/Tutorial/Javascript/README.md]: Nit: capital A in "An element"

**Comment by Cypher1** [comment]
Two suggestions:
1. I don't think this is related to the test failures, so a rebase should bring the tests good.
2. Where possible, if we avoid using `connection = handle` style connection (i.e. those with '=' as the direction arrow) to make connections we'll get better error messages when things do not resolve. This becomes `connection: any handle` in the new syntax which makes it more clear that this lets the particle accept any connection, which may not be what it needs.

Otherwise LGTM (seconding @csilvestrini's suggestions.
